### DISCUSSION TITLE
Timeout obtaining instance API.

### DISCUSSION THREAD

**Discussion by HarithOmari** [pockethost/pockethost#292] [Q&A]
Hi, 
I'm hosting a server for a PocketBase DB, and the server worked well for a couple of days until today, I tried to run the server, and it runs for 5 seconds or so and then it goes back to idle mode, and the database shows me the following error on a blank browser page:

Timeout obtaining instance API.

Can you please help me overcome this error, I'm trying to work.

Please go easy on me with terms and questions, I'm new to all this.

thanks!

**Comment by DiceBreakers** [comment]
I can't help you, but am experiencing a similar thing. Started my project a few days ago and am trying to code the front and back end together today, but am getting the Timeout obtaining instance API message, or sometimes I can get in for a few seconds but then get a "Something went wrong while processing your request" error if I try to take any action.

I know it's only a sample size of two, but doesn't sound isolated. I imagine they'll need to fix something on the back end and we'll be up and running.

**Comment by benallfree** [comment]
Moving to #223 

Thanks for the report @HarithOmari
